fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -1177,4 +1177,25 @@ public void testAppendOnlyToBinary() throws IOException {
         assertEquals(HttpStatus.SC_FORBIDDEN, getStatus(postReq));
     }
 
+    @Test
+    public void testFoafAgent() throws IOException {
+        final String path = ingestObj("/rest/foaf-agent");
+        ingestAcl("fedoraAdmin", "/acls/03/foaf-agent.ttl", path + "/fcr:acl");
+        final String username = "user1";
+
+        final HttpGet req = new HttpGet(path);
+        setAuth(req, username);
```

Comment:
Maybe a comment should be added to the `setAuth()` line noting that actually, no auth headers should be set for this test... that is the point of "acl:agentClass foaf:Agent".